python/mxnet/module/module.py

```diff
@@ -502,6 +502,9 @@ def init_optimizer(self, kvstore='local', optimizer='sgd',
         batch_size = self._exec_group.batch_size
         if kvstore and 'dist' in kvstore.type and '_sync' in kvstore.type:
             batch_size *= kvstore.num_workers
+        if kvstore and 'ngraph' in kvstore.type:
+            batch_size *= kvstore.num_workers
+            optimizer_params['learning_rate'] *= kvstore.num_workers
```

Comment:
Can we leave this as a user-controlled change to learning rate instead of automating it?

Reply:
Sure.